runtime/vara/src/lib.rs

```diff
@@ -568,7 +568,7 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
     type Fallback = onchain::OnChainExecution<OnChainSeqPhragmen>;
     type GovernanceFallback = onchain::OnChainExecution<OnChainSeqPhragmen>;
     type Solver = SequentialPhragmen<AccountId, SolutionAccuracyOf<Self>, ()>;
-    type ForceOrigin = EnsureRoot<AccountId>;
+    type ForceOrigin = AdminOrigin;
```

Comment:
Could you explain please why do we need stacking admin in elections?

Reply:
To control this pallet. Like emergency solution result, min untrusted score, governance fallback

Reply:
As I can see in doc StakingAdmin is only for slashes
https://github.com/gear-tech/gear/blob/e982f460bc17b0fdebd382f899ed80d0c353825c/runtime/vara/src/governance/origins.rs#L37C1-L39

Reply:
updated